net-misc/openssh/files/openssh-9.8_p1-musl-connect.patch

```diff
@@ -0,0 +1,13 @@
+# Pulled patch from Voidlinux
```

Comment:
A link should be included in that case as well as ideally a link to an upstream bug report.

But see https://bugs.gentoo.org/935353#c3 in terms of maybe a better approach.

Reply:
Yea since systemd isn't compatible with musl I can just strip SYSTEMD_NOTIFY instead of this patch